src/components/Card/Card.scss

```diff
@@ -17,11 +17,19 @@
 }
 
 .Header {
-  padding: spacing(loose) spacing(loose) 0;
+  padding: spacing() spacing() 0;
```

Comment:
On https://github.com/Shopify/polaris-react-deprecated/issues/1999#issuecomment-417046795 it was discussed that side paddings should be 16px, but for top and bottom, should it still be 20px?

Reply:
IMO yes, but tagging in @sarahill 👋